README.md

```diff
@@ -9,11 +9,12 @@
 | Debian 8     | [![Docker Pulls](https://img.shields.io/docker/pulls/paulfantom/debian-molecule.svg)](https://hub.docker.com/r/paulfantom/debian-molecule) |
 | Debian 9     | [![Docker Pulls](https://img.shields.io/docker/pulls/paulfantom/debian-molecule.svg)](https://hub.docker.com/r/paulfantom/debian-molecule) |
 | Ubuntu 16.04 | [![Docker Pulls](https://img.shields.io/docker/pulls/paulfantom/ubuntu-molecule.svg)](https://hub.docker.com/r/paulfantom/ubuntu-molecule) |
+| Ubuntu 18.04 | [![Docker Pulls](https://img.shields.io/docker/pulls/paulfantom/ubuntu-molecule.svg)](https://hub.docker.com/r/paulfantom/ubuntu-molecule) |
```

Comment:
Assuming this matrix will expand, it may be best to adjust the names for each image. Perhaps something like ``ubuntu-xenial`` and ``ubuntu-bionic``?

Reply:
I was thinking about naming images like `ubuntu-xenial` or `ubuntu-bionic`, but since `xenial` and `bionic` are just code names for versions I abandoned this idea and used container versioning. This way it is universal across various distros.

Reply:
Also this is the same scheme ubuntu uses